### PR TITLE
DEVX-2314: Update all links from GH repos to docs.confluent.io: full path has changed with docs migration

### PR DESCRIPTION
**Pull request by awalther28**
### Description
https://confluentinc.atlassian.net/browse/DEVX-2314
Update links to map to full path in docs. A few doc links were linked to content that no longer exists. In those cases I picked a document that best fits, in one case I removed the link altogether. 

I found the docs links by using `grep -R " https://docs.confluent.io/current" .`. Please note if you think that is not sufficient. 

No code related to tests changes in this PR!

### Staging Docs
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/DEVX-2314

### CODE REVIEW

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/persistent-query.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
 
-It should not escape your notice that we are turning a stream into a table. A table is always the result of using https://docs.confluent.io/current/ksql/docs/developer-guide/create-a-table.html#create-a-ksql-table-from-a-ksql-stream[the GROUP BY clause] on a stream. As we noted in the previous step, we are also computing an aggregate over the grouped values with `SUM(TICKET_TOTAL_VALUE)`. This function creates a new column in the resulting table, which we give a readable name using the `AS TOTAL_VALUE` clause.
+It should not escape your notice that we are turning a stream into a table. A table is always the result of using the GROUP BY clause on a stream. As we noted in the previous step, we are also computing an aggregate over the grouped values with `SUM(TICKET_TOTAL_VALUE)`. This function creates a new column in the resulting table, which we give a readable name using the `AS TOTAL_VALUE` clause.
```
Comment (ybyzek) [inline _includes/tutorials/aggregating-sum/ksql/markup/dev/persistent-query.adoc:3]: @awalther28 should there be backticks here?

```suggestion
It should not escape your notice that we are turning a stream into a table. A table is always the result of using the `GROUP BY` clause on a stream. As we noted in the previous step, we are also computing an aggregate over the grouped values with `SUM(TICKET_TOTAL_VALUE)`. This function creates a new column in the resulting table, which we give a readable name using the `AS TOTAL_VALUE` clause.
```

**Review by ybyzek — APPROVED** [review]
@awalther28 great fixes, LGTM (pending one minor comment in review)

File: _includes/tutorials/aggregating-sum/ksql/markup/dev/persistent-query.adoc (hunk 1 above)
Comment (awalther28) [reply _includes/tutorials/aggregating-sum/ksql/markup/dev/persistent-query.adoc:3]: yes 💯

**Review by bbejeck — APPROVED** [review]
LGTM!

**Pull request merged**